react/week2/exersice-react/src/components-todo/TodoItems.js

```diff
@@ -0,0 +1,23 @@
+function TodoItems() {
+const doItems = [
+        {
+          id: 1,
+          description: 'Get out of bed',
+          date: 'Wednesday September  13, 2021',
+        },
+        {
+          id: 2,
+          description: 'Brush teeth',
+          date: 'Thursday September  14, 2021',
+        },
+        {
+          id: 3,
+          description: 'Eat breakfast',
+          date: 'Friday September  15, 2021',
+        }
+      ];
+
+    return doItems;
+}
+
+export default TodoItems;
```

Comment:
You may better turn this file into a json file instead of using a function that makes an extra code line that is not needed.

```suggestion
[
  {
    "id": 1,
    "description": "Get out of bed",
    "date": "Wednesday September  13, 2021"
  },
  {
    "id": 2,
    "description": "Brush teeth",
    "date": "Thursday September  14, 2021"
  },
  {
    "id": 3,
    "description": "Eat breakfast",
    "date": "Friday September  15, 2021"
  }
]

```
and file name can be something like todoList.json